TargetLibraries/Snitch/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+file(GLOB_RECURSE SOURCES
+  "src/**"
+)
+
+include(cmake/snitch-runtime-precompiled.cmake)
+
+add_deeploy_library(deeploysnitch STATIC ${SOURCES})
+target_include_directories(deeploysnitch
+  PUBLIC
+  ${CMAKE_CURRENT_LIST_DIR}/inc
+)
+
+target_compile_options(deeploysnitch PRIVATE
+  -Wno-implicit-function-declaration
+  -Wno-implicit-int-conversion
+  -Wno-sign-conversion
+  -Wno-sign-compare
```

Comment:
is discarding these warnings necessary? this is the code written and maintained by us and these warnings could prevent bugs.

Reply:
@Xeratec Is there any reason for these warning suppression (like polluting the logs)?

Reply:
I think they could be removed. Back then, I ported the flags in the Make flow one to one. @tahaelbayad Can you try to remove them and check if there are warnings?

Reply:
Yes I will do it

Reply:
Since we are not pedantic with our warnings, removing the flags should be fine.
From what taha reported, it's mostly issues with types for the chunk calculation.
From talking with Victor, I understood this PR is blocking for further development so I would suggest we make an issue and leave the fix for later.
@tahaelbayad could you file an issue with the log of the warnings that are produced?